### PR TITLE
Adds import-mode feature to 4.13 release notes

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5368

Link to docs preview:
https://55972--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-oc

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -59,6 +59,20 @@ This release adds improvements related to the following components and concepts.
...
+==== oc
+
+With {product-title} {product-version}, a new `oc` command line interface (CLI) flag, `--import-mode`, has been added to the following `oc` commands:
+
+* `oc import-image`
+* `oc tag`
+* `oc new-build`
+* `oc new-app`
+
+With this enhancement, users can set their `importMode` to `Legacy` or `PreserveOriginal`. This provides users the option to import all manifests of a manifest list when running the aforementioned `oc` commands.
```
Comment (flavianmissi) [inline release_notes/ocp-4-13-release-notes.adoc:72]: Do you think it's worth tying this up up multi-arch support somehow?

**Comment by flavianmissi** [comment]
looking good!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -59,6 +59,20 @@ This release adds improvements related to the following components and concepts.
...
+==== --import-mode enhancement for OpenShift CLI (oc)
+
+As of {product-title} 4.12, limited usability (Technology Preview) support for manifest listed images on image streams is available.
+
+With {product-title} {product-version}, a new `oc` command line interface (CLI) flag, `--import-mode`, has been added to the following `oc` commands:
+
+* `oc import-image`
+* `oc tag`
+
+With this enhancement, users can set the `--import-mode` flag to `Legacy` or `PreserveOriginal`, which provides users the option to import a single sub-manifest, or all manifests, of a manifest list when running the aforementioned `oc` commands.
```
Comment (JoeAldinger) [inline release_notes/ocp-4-13-release-notes.adoc:72]:
```suggestion
With this enhancement, users can set the `--import-mode` flag to `Legacy` or `PreserveOriginal`, which provides users the option to import a single sub-manifest, or all manifests, of a manifest list when running the new`oc` commands.
```

Comment (JoeAldinger) [reply release_notes/ocp-4-13-release-notes.adoc:72]: I would either cut it entirely or maybe go with new for minimalism.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -59,6 +59,20 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-oc"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-12-import-image"]
```
Comment (JoeAldinger) [inline release_notes/ocp-4-13-release-notes.adoc:62]: Should this id be ocp-4-13-import-image?  The 4.12 in 4.13 RNs has me a bit confused.

**Review by JoeAldinger — COMMENTED** [review]
small suggestion and questions. Otherwise, lgtm.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
Comment (stevsmit) [reply release_notes/ocp-4-13-release-notes.adoc:72]: Conflicted here. Those aren't the new commands. The --import-mode is an enhancement _to_ those commands, and has to be run with them.

**Review by michaelryanpeter — COMMENTED** [review]
A couple of suggestions to consider, but no blockers.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**